src/types/chat/kind.rs

```diff
@@ -14,6 +14,11 @@ pub enum Kind {
         first_name: String,
         /// The last name of the user.
         last_name: Option<String>,
+        /// The Bio of the other party in a private chat. Returned only
+        /// in [`GetChat`].
+        ///
+        /// [`GetChat`]: ../../methods/struct.GetChat.html
```

Comment:
```suggestion
        /// [`GetChat`]: crate::methods::GetChat
```

Since intra-doc links were stabilized in 1.48, please use them.